### ISSUE TITLE
Still active 

### ISSUE THREAD

**Issue by daslicht** [binaryjs/binaryjs#118]
Is this Project Still active  ?

**Comment by soleilnoirmedia-deprecated** [comment]
No very likely. Last commit was merged was about 8 months ago or so. I commented that if the project is not being maintained maybe it could be transferred to somebody else. It seems that while development has stopped, usage has not.

**Comment by niieani** [comment]
Could @ericz or @dominicklim comment whether they plan on continuing work on this? The domain name seems to have been lost. Perhaps it would be good if this project were transferred to another interested maintainer -- or somebody could just fork this (it's MIT after all)? It has a lot of potential.

**Comment by phutchins** [comment]
I'm considering forking as I like the idea of it but have some blocking issues on the client side... Doesn't look like the current owner is going to chime in. Wonderful project but it needs some love. :)

**Comment by niieani** [comment]
If you fork, let us know here. Some of us could help maintaining.

**Comment by phutchins** [comment]
I went ahead and forked here: https://github.com/phutchins/binaryjs

I would love some experienced and interested maintainers to join me in keeping this up to date. @niieani I will go ahead and add you as a maintainer. If you have any other suggestions for maintainers, let me know.

**Comment by niieani** [comment]
You should also fork https://github.com/binaryjs/streamws since that's a dependency. It's basically `ws` with a few changes; it would probably be simpler to port these changes to a fresh fork of `ws` though since it's so outdated. It might even be that a fork isn't required now, but that it's possible to achieve the same functionality with the original.

**Comment by niieani** [comment]
And then there are also BinaryPack libs: https://github.com/binaryjs

Maybe we should create a new GitHub organization which contains all those forks? We'd need a new name though, any ideas?
I'd be great to also convert the code-base to something more maintainable. I'd prefer TypeScript, but pure ES2015 would be a good start if you're not experienced in TS.

**Comment by phutchins** [comment]
My goals are to make streaming in the browser and binary as easy as it is in nodejs. Sounds like this is exactly what I need and would be great to contribute back keeping all of these projects alive.

For names, we could stick it under PiPo, which is short for Ping Pong. PiPo is a project that I started to build a client side encrypted chat client/server similar to Slack but encrypted and secure. It uses all of these concepts and might be a good place to keep the projects. If it makes more sense to put them under their own org that works for me also...

**Comment by phutchins** [comment]
@niieani PiPo was taken as a username so I just went ahead and created an org called awesomesocks. We can change it if we like but I wanted to go ahead and get this going :).

**Comment by phutchins** [comment]
Link here: https://github.com/awesomesocks

@niieani, your invite is pending...

I feel like we should get binaryjs up to date and then discuss the plan going forward.

I have another library for streaming from HTML5 file objects that I'll probably move over to this org also as it seems to fit in.

**Comment by niieani** [comment]
@phutchins I'm a member without access though. Cannot push to any of the repos.

**Comment by bradisbell** [comment]
Did anyone end up adopting BinaryJS?  The forks listed here appear to be the same as the main?

**Comment by niieani** [comment]
@bradisbell Me and @phutchins adopted, but haven't had the time to work on this yet (I've been busy with other stuff). If you want to join the team, @phutchins can add you.

**Comment by phutchins** [comment]
@bradisbell Yes, we did. It is active and ready to go. If you have changes or PR's that you would like to propose, submit issues/pr's and well review and get it merged if it looks good!
